### PR TITLE
css fixes

### PR DESCRIPTION
**Pull request by JaneHazen**

### CODE REVIEW

File: src/scripts/clipperUI/componentBase.ts [hunk 1]
```diff
@@ -158,6 +158,12 @@ export abstract class ComponentBase<TState, TProps> {
 					element.style.outlineStyle = "";
 				}
 			},
+			onkeydown: (e: KeyboardEvent) => {
+				if (e.which === Constants.KeyCodes.space) {
```
Comment (mttwc) [inline src/scripts/clipperUI/componentBase.ts]: If you press space in a text box, would this prevent the space character from being inputted?

File: src/styles/clipper.less [hunk 2]
```diff
@@ -76,14 +76,13 @@
 }
 
 #closeButtonContainer {
-	position: absolute;
+	float: right;
```
Comment (mttwc) [inline src/styles/clipper.less:79]: What does this do?

File: src/styles/mainClassDefinitions.less [hunk 3]
```diff
@@ -467,8 +461,8 @@
 	border-radius: 4px;
 	text-align: center;
 	margin-bottom: 10px;
-	margin-left: 5px;
 	margin-right: 5px;
+	width: 100%;
```
Comment (mttwc) [inline src/styles/mainClassDefinitions.less]: Is this for the clip button? Wouldn't leave in the margin-left cause this to not take up all the space on the right?

**Review by mttwc — CHANGES_REQUESTED** [review]

File: src/styles/clipper.less (hunk 2 above)
Comment (JaneHazen) [reply src/styles/clipper.less:79]: Using absolute positioning makes the CSS a mess for all of its nested elements, so I just floated this to the right to get it in the right position without making the closeButton and closeButtonIcon do crazy thing

File: src/scripts/clipperUI/componentBase.ts (hunk 1 above)
Comment (JaneHazen) [reply src/scripts/clipperUI/componentBase.ts]: That still works fine- when it is in the textbox it still allows spaces and enter for paragraphs, this just directs you into that box

File: src/styles/mainClassDefinitions.less (hunk 3 above)
Comment (JaneHazen) [reply src/styles/mainClassDefinitions.less]: I'm not sure what you mean? The margin-left was causing this to pop out of alignment with the location label and the picker

Comment (mttwc) [reply src/styles/mainClassDefinitions.less]: As discussed, we should be able to remove margin-right

File: src/scripts/clipperUI/componentBase.ts (hunk 1 above)
Comment (mttwc) [reply src/scripts/clipperUI/componentBase.ts]: Confirmed working

File: src/styles/mainClassDefinitions.less [hunk 4]
```diff
@@ -467,8 +461,8 @@
 	border-radius: 4px;
 	text-align: center;
 	margin-bottom: 10px;
-	margin-left: 5px;
```
Comment (mttwc) [inline src/styles/mainClassDefinitions.less:470]: As discussed, when this comes into focus, the UI shifts a little.

**Review by mttwc — CHANGES_REQUESTED** [review]

File: src/styles/mainClassDefinitions.less [hunk 5]
```diff
@@ -462,7 +462,7 @@
 	text-align: center;
 	margin-bottom: 10px;
 	margin-right: 5px;
-	width: 100%;
+	margin-left: 5px;
```
Comment (mttwc) [inline src/styles/mainClassDefinitions.less]: As discussed, we need to consult with design to see if we'd rather have missing horizontal focus borders or misaligned elements. Either way, we need to deal with a tradeoff and we try not to make these decisions independently as they will no longer match redlines.

**Review by mttwc — CHANGES_REQUESTED** [review]

**Review by mttwc — APPROVED** [review]

**Pull request merged**